docs/scenario_guidelines/_index.md

```diff
@@ -39,26 +39,39 @@ Rather you are working on a new scenario or updating an existing one, a scenario
 
 * As always, we are here for you. In your scenario "Feature request" issue, tag one of the maintainers and we will answer any question you may have.
 
+## Folder Structure
+
+The Azure Arc Jumpstart repository follows a specific folder structure that you should get familiar with before creating a new scenario.
+
+* Automations, scripts, templates, json files, etc. should be placed under each Azure Arc pillar that corresponds to each scenario.
+* The guide for the scenario should be under docs and follow a similar structure.
+
+![Screenshot of folder structure](./03.png)
+
 ## Indexing
 
-The Azure Arc Jumpstart website is using [HUGO](https://gohugo.io/) as its web framework alongside [Docsy](https://www.docsy.dev/) as it theme of choice.
+The Azure Arc Jumpstart website is using [HUGO](https://gohugo.io/) as its web framework alongside [Docsy](https://www.docsy.dev/) as its theme of choice.
 
 * Scenarios must be created in its respected "docs" folder in the project GitHub repository and with the "_index.md_" filename for it to get published.
 
-    ![Screenshot of index file location](./03.png)
+    ![Screenshot of index file location](./04.png)
 
 * Both the "_title_" and the "_linkTitle_" must be the same.
 
-    ![Screenshot of matching link and linkTitle](./04.png)
-
-* The "_Weight_" felid number represents the location of the scenario comparing to other scenarios on the same page on the website.
+    ![Screenshot of matching link and linkTitle](./05.png)
 
-    ![Screenshot of weight numbering](./05.png)
+* The "_Weight_" field number represents the location of the scenario comparing to other scenarios on the same page on the website.
 
     ![Screenshot of weight numbering](./06.png)
 
+    ![Screenshot of weight numbering](./07.png)
+
 * As always, if you are not sure and you have questions about this section, we recommend looking at other published scenarios and/or reaching out to one of the maintainers.
 
+## Description
+
+All Jumpstart scenarios start with an overview of what the outcome will be after you run the automation. This description should also mention the starting point for the automation, for example for day two scenarios you may need an already deployed server or Kubernetes cluster that is onboarded onto Azure Arc, you should also include pointers to scenarios that would allow you to get to that starting point.
```

Comment:
```suggestion
All Jumpstart scenarios start with an overview of what the outcome will be after you run the automation. This description should also mention the starting point for the automation, for example for "Unified Operations" (day-2) scenarios you may need an already deployed server or Kubernetes cluster that is onboarded onto Azure Arc, you should also include pointers to scenarios that would allow you to get to that starting point.
```